src/WMStream.ts

```diff
@@ -0,0 +1,29 @@
+const util = require('util');
+const stream = require('stream');
+const Writable = stream.Writable;
+
+export default class WMStream extends Writable 
```

Comment:
Please rename class name. Not sure what "WM" stands for? Also I am surprised linter isn't complaining about the lack of documentation of an exported class?